.projenrc.ts

```diff
@@ -0,0 +1,35 @@
+import { JsiiProject } from 'projen/lib/cdk';
+import { BundleKics } from './projenrc';
+
+const project = new JsiiProject({
+  author: 'Checkmarx',
+  authorAddress: 'REPLACEME@checkmarx.com',
+  defaultReleaseBranch: 'main',
+  devDeps: [
+    'cdklabs-projen-project-types',
+    '@octokit/types',
+    '@octokit/rest',
+    'mock-fs',
+    '@types/mock-fs',
+    'fs-extra',
+    '@types/fs-extra',
+  ],
+  name: 'kics-cdk-validator-plugin',
+  projenrcTs: true,
+  release: false,
```

Comment:
Flipping this to true will make projen generate a new workflow file, with actions to pulish releases to GitHub and to NPM.